test/uk/gov/hmrc/transitmovementsrouter/generators/TestModelGenerators.scala

```diff
@@ -123,4 +131,49 @@ trait TestModelGenerators extends BaseGenerators {
     )
   }
 
+  implicit def arbitrarySdesResponse(): Arbitrary[SdesNotificationItem] = Arbitrary {
+    for {
+      filename          <- Gen.alphaNumStr
+      correlationId     <- Gen.alphaNumStr
+      checksum          <- Gen.stringOfN(4, Gen.alphaChar)
+      checksumAlgorithm <- Gen.alphaNumStr
+      notification = SdesNotification.FileProcessed
+      received     = Instant.now()
+      properties   = Seq(SdesProperties("x-conversation-id", "123e4567-e89b-12d3-a456-426614174000"))
+    } yield SdesNotificationItem(
+      notification,
+      filename,
+      correlationId,
+      checksum,
+      checksumAlgorithm,
+      received,
+      None,
+      received,
+      properties
+    )
+  }
+
+  implicit def arbitrarySdesFailureResponse(): Arbitrary[SdesNotificationItem] = Arbitrary {
+    for {
+      filename          <- Gen.alphaNumStr
+      correlationId     <- Gen.alphaNumStr
+      checksum          <- Gen.stringOfN(4, Gen.alphaChar)
+      checksumAlgorithm <- Gen.alphaNumStr
+      notification = SdesNotification.FileProcessingFailure
```

Comment:
in both cases (FileProcessed and FileFailed) we need to return 200 so better to use above one and remove this.